hive-metastore/src/test/java/org/apache/iceberg/hive/TestHiveCommits.java

```diff
@@ -18,6 +18,7 @@
  */
 package org.apache.iceberg.hive;
 
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
```

Comment:
why are we switching this one to a static import? Same for the other files. I'm ok doing that as long as the diff isn't large, but I would assume that we'd do the same for `assertThat()`

Reply:
There are very less references for assertThatThrownBy compared to assertThat

Reply:
not sure I'm following. There's currently no `assertThat()` usage in this class, so we can just add a static import for it

Reply:
Added.